src/last_sign_state.rs

```diff
@@ -1,31 +1,25 @@
 use abscissa::Error;
+use atomicwrites::{AtomicFile, OverwriteBehavior};
 use serde_json;
 use std::{
     fmt::{self, Display},
-    fs::{File, OpenOptions},
-    io::prelude::*,
-    path::Path,
+    fs,
+    io::{self, prelude::*},
+    path::{Path, PathBuf},
 };
 use tendermint::{block, chain};
 
-#[derive(Serialize, Deserialize)]
+#[derive(Clone, Debug, Default, Serialize, Deserialize)]
 struct LastSignData {
     pub height: i64,
     pub round: i64,
     pub step: i8,
     pub block_id: Option<block::Id>,
 }
 
-const EMPTY_DATA: LastSignData = LastSignData {
-    height: 0,
-    round: 0,
-    step: 0,
-    block_id: None,
```

Comment:
Given these are all the "zero value" this can be replaced with `#[derive(Default)]`